openedx/core/djangoapps/safe_sessions/middleware.py

```diff
@@ -54,6 +54,10 @@
 SSL-protected channel.  Otherwise, a session hijacker could copy
 the entire cookie and use it to impersonate the victim.
 
+Custom Attributes:
+    safe_sessions.user_mismatch: 'request-response-mismatch' | 'request-session-mismatch'
```

Comment:
Would it be helpful to have the user ids in another attribute as well?

Reply:
I didn't want to put the user ids in NR if I don't have to.  Since they're in splunk still I can always get them there if I need to.